stock_product_qty_by_packaging/models/stock_move_line.py

```diff
@@ -10,4 +10,4 @@ class StockMoveLine(models.Model):
     _name = "stock.move.line"
     _inherit = ["stock.move.line", "product.qty_by_packaging.mixin"]
 
-    _qty_by_pkg__qty_field_name = "product_qty"
+    _qty_by_pkg__qty_field_name = "qty_done"
```

Comment:
This is not the same thing. Please check move line fields https://github.com/OCA/OpenUpgrade/blob/de3714a13e5ef57b5abbb2e30ed5080183973cfa/openupgrade_scripts/scripts/stock/16.0.1.1/upgrade_analysis.txt#L55-L60

Reply:
hi @simahawk , you're right, it's my bad, I will replace it by `reserved_qty` in `16.0`